src/bentoml/grpc/v1/service.proto

```diff
@@ -43,9 +43,9 @@ message ServiceMetadataResponse {
     // name is the name of the API.
     string name = 1;
     // input is the input descriptor of the API.
-    DescriptorMetadata input = 2;
+    optional DescriptorMetadata input = 2;
```

Comment:
Isn't this a breaking change? Can we do this?

Reply:
optional is not a breaking change right? it would still be there.

Reply:
Hmm, but doesn't this break old clients?

Reply:
not necessary right? If the old client specify input then it still works right?